### DISCUSSION TITLE
Autoscaling GoodJob queues

### DISCUSSION THREAD

**Discussion by adamlogic** [bensheldon/good_job#839] [Show and tell]
I just published the `judoscale-good_job` gem to integrate GoodJob with the Judoscale autoscaler. I wrote up a post on [how to autoscale your GoodJob queues](https://judoscale.com/updates/autoscale-good-job-queues) that folks might find useful if they're running on Heroku.

If you have questions or suggestions, I'd love to hear them!

**Comment by bradgessler** [comment]
That link 404's, but I am interest in a job autoscaler running on Fly.io with Fly Machines. If that's something you're interested in getting running LMK!

**Reply by bensheldon** [reply]
That would be rad! I just merged a minute ago and will momentarily release:

- https://github.com/bensheldon/good_job/pull/1159

to allow [this nifty thing on Fly.io](https://github.com/bensheldon/good_job/issues/1140#issuecomment-1799308770).

I have a lot of interest, but not a lot of time, in supporting autoscaling on different platforms. So mainly if someone can give me the stats/metric to expose, and the various methods to expose it, I can run with that.

**Reply by adamlogic** [reply]
Thanks for the heads-up about that link! It's fixed now, but probably not helpful to you since we haven't integrated with Fly yet.

We've had several folks ask us about autoscaling on Fly, so it's definitely on our radar. I'll post back here when I have an update.
